.eslintrc.cjs

```diff
@@ -79,7 +79,7 @@ module.exports = {
     '@typescript-eslint/consistent-indexed-object-style': 'error',
     '@typescript-eslint/consistent-type-assertions': [
       'error',
-      { assertionStyle: 'angle-bracket', objectLiteralTypeAssertions: 'never' },
+      { assertionStyle: 'as', objectLiteralTypeAssertions: 'never' },
```

Comment:
![image](https://github.com/user-attachments/assets/e3ab13de-7605-41d1-b9a7-50055b452c08)

For posterity.